src/shared/ui/button/button.tsx

```diff
@@ -0,0 +1,21 @@
+import { clsx } from 'clsx'
+
+import type { ButtonHTMLAttributes } from 'react'
+
+interface Props extends ButtonHTMLAttributes<HTMLButtonElement> {
+  variant?: 'primary'
+}
+
+export default function Button({ children, variant = 'primary', className, ...props
+}: Props) {
+  const baseStyle = 'w-24 h-10'
+  const variantStyle = {
+    primary: 'bg-blue-500 text-white'
+  }
+
+  return (
+    <button className={
+      clsx(
+        baseStyle, variantStyle[variant], className)}{...props}>{children}</button>
```

Comment:
여기 prettier가 적용된건가요? 읽기가 힘드네요..

```suggestion
        baseStyle, variantStyle[variant], className)}{...restProps}>{children}</button>
```

restProps로 작성해주는게 나머지 프롭스라는 뜻으로 명확할 것 같아요.